README.md

```diff
@@ -39,24 +39,22 @@ and run `cd pollen_py && flit install -s --user`. You will need [`flit`][flit].
 
 #### Calyx
 
-Follow these [instructions](https://docs.calyxir.org/) to install Calyx. You must complete the [first](https://docs.calyxir.org/#installing-from-source-to-use-and-extend-calyx) and [third](https://docs.calyxir.org/#installing-the-command-line-driver) sections, installing Calyx from source, but feel free to skip the second section. Then, from the root of the Calyx repository, install the Calyx interpreter by running 
-```
-cargo build -p interp
-```
-
-The last step should be running `fud check`, which will report that some tools are unavailable. This is okay for our purposes.
-
-After completing the above, run
-```
-fud config stages.calyx.exec <full path to calyx repository>/target/debug/calyx
-fud config stages.interpreter.exec <full path to calyx repository>/target/debug/interp
-fud check
-```
+Below we show you how to build Calyx from source and set it up for our use.
+We suggest you make `calyx` and `pollen` sibling directories.
```

Comment:
Maybe we can preserve the links to the Calyx docs here too, just so someone who wants more details can find them?